config/options.go

```diff
@@ -112,6 +112,10 @@ type Options struct {
 	// used for authentication requests and callbacks
 	AuthenticateURLString string   `mapstructure:"authenticate_service_url" yaml:"authenticate_service_url,omitempty"`
 	AuthenticateURL       *url.URL `yaml:"-,omitempty"`
+	// AuthenticateURL represents the externally accessible http endpoints
```

Comment:
I think you forgot to update the comment here?

Reply:
Ops, my bad. Fixed.